.github/workflows/docker.yml

```diff
@@ -29,6 +29,6 @@ jobs:
       registry: docker-oss.nexus.famedly.de
       image_name: ansible
       tags: |
-        py-${{ matrix.python_base_version }}-ansible${{ matrix.ansible_version }}
+        py-${{ matrix.python_base_version }}-ansible-${{ matrix.ansible_version }}
```

Comment:
Consistency with what we had on gitlab would be to remove the dash after py instead: https://github.com/famedly/container-image-ansible/blob/main/.gitlab-ci.yml#L13C142-L13C142

I'm fine with it either way though

Reply:
hm, I feel like having the dash is more readable, so I'd like to take the opportunity of the migration to just make that improvement and keep it consistent with that in the future